README.md

```diff
@@ -28,6 +28,21 @@ The site is published automatically by [Netlify](https://www.netlify.com/) whene
 
 We strongly encourage you to contribute to this site! For more information, see the [contributing](CONTRIBUTING.md) guide.
 
+### Publishing new Jaeger version
+
+Each Jaeger version is documented in a separate directory e.g. [content/docs/1.8](./content/docs//1.8).
+
+When submitting changes for the new version please split commits with changes
+and files copy of the previous documentation files. The following steps shows
+a possible workflow:
```

Comment:
"When submitting changes for a new version of Jaeger, please first make a copy of the current version's documentation under the new version number and commit it. Then apply any relevant changes to update the documentation for the new version as a separate PR, to make it easier to review the changes. A possible workflow is:"